workorder-recheck.lua

```diff
@@ -3,18 +3,66 @@
 
 workorder-recheck
 =================
-Sets the status to ``Checking`` (from ``Active``) of the selected work order (in the ``j-m`` or ``u-m`` screens).
+Sets the status to ``Checking`` (from ``Active``) of the selected work order.
 This makes the manager reevaluate its conditions.
 ]====]
```

Comment:
docs have moved to docs/workorder-recheck.rst. could you make the edits there? Also, please remove the "unavailable" tag in the rst file since now the script is available : P

Please also remove the docstring here in this file -- it's left over from the .rst migration and will just get out of sync with the real docs.